common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -253,6 +253,86 @@ def test_timed_exam_flow(self):
         # Then I am taken to the exam with a timer bar showing
         self.assertTrue(self.courseware_page.is_timer_bar_present)
 
+    def test_time_allotted_field_is_not_visible_with_none_exam(self):
```

Comment:
These tests (that are only using Studio) really belong over in the Studio acceptance tests (test/acceptance/studio).